docker/Dockerfile-proxy

```diff
@@ -7,13 +7,15 @@ ENV FRP_VERSION=v0.51.3
 RUN apt-get update
 RUN apt-get install -y curl
 
-RUN addgroup -S frp \
-    && adduser -D -S -h /var/frp -s /sbin/nologin -G frp frp \
-    && curl -fSL https://github.com/fatedier/frp/releases/download/${FRP_VERSION}/frp_${FRP_VERSION:1}_linux_amd64.tar.gz -o frp.tar.gz \
-    && tar -zxv -f frp.tar.gz \
-    && rm -rf frp.tar.gz \
-    && mv frp_*_linux_amd64 /frp \
-    && chown -R frp:frp /frp
+SHELL ["/bin/bash", "-c"]
```

Comment:
Unrelated change that helps run frp in api; doesn't break logic as it's only for local development